src/workers/notifications/squadPostModerationRejected.ts

```diff
@@ -0,0 +1,31 @@
+import { generateTypedNotificationWorker } from './worker';
+import { NotificationType } from '../../notifications/common';
+import { NotificationPostModerationContext } from '../../notifications';
+import { User } from '../../entity';
+import { SquadPostModerationStatus } from '../../entity/SquadPostModeration';
+
+const worker =
+  generateTypedNotificationWorker<'api.v1.squad-post-moderation-submitted'>({
+    subscription: 'api.v1.squad-post-moderation-rejected-notification',
+    handler: async ({ post }, con) => {
+      if (
+        !post?.createdById ||
+        post.status !== SquadPostModerationStatus.Rejected
+      ) {
+        return;
+      }
+
+      await con
+        .getRepository(User)
+        .findOneOrFail({ where: { id: post.createdById } });
```

Comment:
Just to fail if user doesn't exist?

Reply:
Maybe better to query and early return since this will log errors from notification worker

Reply:
Yeah, let's do early return instead.